dockers/docker-orchagent/ndppd.conf.j2

```diff
@@ -0,0 +1,38 @@
+{% block banner %}
+# =========== Managed by sonic-cfggen -- DO NOT edit manually! ====================
+# Generated by /usr/share/sonic/templates/ndppd.conf.j2 using config DB data
+# File: /etc/ndppd.conf
+#
+{% endblock banner %}
+# Config file for ndppd, the NDP Proxy Daemon
+# See man page for ndppd.conf.5 for descriptions of all available options
+#
+{%  if VLAN_INTERFACE and VLAN_INTERFACE|pfx_filter|length > 0%}
+{#      Get all VLAN interfaces that have proxy_arp enabled #}
+{%      set proxy_interfaces = {} %}
+{%      for intf in VLAN_INTERFACE %}
+{%          if "proxy_arp" in VLAN_INTERFACE[intf] and VLAN_INTERFACE[intf]["proxy_arp"] == "enabled" %}
+{%              set _x = proxy_interfaces.update({intf: []}) %}
+{%          endif %}
+{%      endfor -%}
+
+{#      Add each IPv6 prefix from each proxy_arp interface #}
+{%      for (intf, prefix) in VLAN_INTERFACE|pfx_filter %}
+{%          if intf in proxy_interfaces and prefix | ipv6 %}
+{%              set _x = proxy_interfaces[intf].append(prefix) %}
+{%          endif %}
+{%      endfor -%}
+
+{%      for intf, prefix_list in proxy_interfaces.items() %}
+{%          if prefix_list %}
+
+proxy {{ intf }} {
+{%              for prefix in prefix_list %}
+    rule {{ prefix | network }}/{{ prefix | prefixlen }} {
+        static
+    }
```

Comment:
If there are multiple IPv6 prefixes, are there any separators required or just appending rule is enough?

Reply:
Just appending the rule works (no commas needed between rules), unfortunately the format is not standard JSON.